torch/testing/_internal/common_utils.py

```diff
@@ -63,6 +63,7 @@
 if sys.platform == 'win32':
     FILE_SCHEMA = "file:///"
 
+# Environment variable `IN_CI` is set in `.jenkins/common.sh`.
 IS_IN_CI = os.getenv('IN_CI') == '1'
```

Comment:
can we unify `IN_CI` vs `IS_IN_CI` as well?

Reply:
IMO, it's probably fine b/c
1. most of the other python var like this in common_utils have different/shorter name than env var
2. it's easier to find usage if they are different through code search
what do you think?

Reply:
But why would we want them to be different though? Is it just so one could be seen as a global var where as the other would be an environment variable?